capellambse/model/common/accessors.py

```diff
@@ -1769,9 +1769,7 @@ def __set__(
                     "Moving model objects is not supported yet"
                 )
             if self.__get__(obj) is not None:
-                raise NotImplementedError(
-                    "Replacing model objects is not supported yet"
-                )
+                obj._element.remove(obj._element.find(self.role_tag))
```

Comment:
The call to `self.__get__()` will give us the constructed proxy for the existing element. We can use its return to avoid the extra `find()`.